### PR TITLE
Fix minmax wizardlet selection

### PR DESCRIPTION
**Pull request by lchanmann**
## Ref
- https://nanopay.atlassian.net/browse/NP-7603
- https://nanopay.atlassian.net/browse/NP-6448

## Changes
- Add adaptArrayElement to StringArray
- Fix minmax wizardlet selection

### CODE REVIEW

**Review by KernelDeimos — APPROVED** [review]

File: src/foam/nanos/crunch/MinMaxCapabilityData.js [hunk 1]
```diff
@@ -31,21 +31,9 @@ foam.CLASS({
...
-        }
-
-        if ( foam.Function.isInstance(n.toString) ) {
-          return [
-            n.toString()
-          ];       
-        }
-
-        return n;
+      adaptArrayElement: function(o, prop) {
```
Comment (tharmaman) [inline src/foam/nanos/crunch/MinMaxCapabilityData.js:34]: So the adaptArrayElement defined in types always returns a string?

**Comment by tharmaman** [comment]
So this fixes the min max wizardlet save issue?

**Comment by lchanmann** [comment]
@tharmaman yes, it fixes the minmax selection which is causing the availability of the prereqs and send them to cancelSave_ instead of save_.

File: src/foam/nanos/crunch/MinMaxCapabilityData.js (hunk 1 above)
Comment (lchanmann) [reply src/foam/nanos/crunch/MinMaxCapabilityData.js:34]: yes the element of StringArray should always be string.

**Pull request merged**